src/pages/Login.tsx

```
@@ -0,0 +1,123 @@
+    try {
+      const response = await login({ mail: email, password });
+
+      const sessionId = response.sessionId || response.token;
+      if (sessionId) {
+        cookie.set("SESSION_ID", sessionId, { path: "/", expires: 1 });
+        isLogin();
+        navigate("/jobs");
+      } else {
+        alert("세션 정보를 받아오지 못했습니다.");
+      }
+    } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)